node_modules/oae-authentication/tests/test-external-strategies.js

```diff
@@ -1293,4 +1293,165 @@ describe('Authentication', function() {
             }, callback);
         });
     });
+
+    describe('Google authentication', function() {
+
+        /**
+         * Configure a set of allowed domains with the `localhost` tenant's google authentication
+         *
+         * @param  {String}         domains     The comma-separated list of allowed domains
+         * @param  {Function}       callback    Standard callback function
+         */
+        var setGoogleDomains = function(domains, callback) {
+            var configUpdate = {};
+            configUpdate['oae-authentication/google/domains'] = domains;
+            ConfigTestUtil.updateConfigAndWait(globalAdminRestContext, global.oaeTests.tenants.localhost.alias, configUpdate, function(err) {
+                assert.ok(!err);
+            });
+
+            // Wait until the authentication api has finished refreshing its strategies
+            AuthenticationAPI.once(AuthenticationConstants.events.REFRESHED_STRATEGIES, function(tenant) {
+                return callback();
+            });
+        };
```

Comment:
Can we wrap this up into a `oae-authentication/lib/test/util.js` file, with a method name such as:

`AuthenticationTestUtil.assertUpdateAuthConfigSucceeds`. It would then take care of both the `updateConfigAndWait`, `AuthenticationAPI.once...`.

Additionally we can add some basic assertions in the generic method such as getting the configuration point afterward and ensuring it is the value we set it to whenever it is used.